cmsController.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+
+function callDatabase() {
+    $db = new PDO('mysql:host=127.0.0.1; dbname=portfolioKyam', 'root');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+$db = callDatabase();
```

Comment:
your doing this at the top of every file that includes this file so your now doing it twice...